### PR TITLE
Prevent re-render of PromotedAddonsCard

### LINKED ISSUE #9737: Sponsored add-on impressions are recorded twice with SSR
Impression events are being sent to GA twice when the homepage is rendered on the server. If a user lands on a different page, and navigates to the homepage, it sends the expected number of events, but if a user starts on the homepage, or refreshes the homepage, then two events are sent for each add-on.

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #9737 

This prevents the double-render of the `PromotedAddonsCard`, which was causing two sets of events to be dispatched. We're still not sure why this issue only appears with SSR, but this is an acceptable fix for the problem.

I am going to expand on this to move the logic to fetch the `addons` into the `PromotedAddonsCard` as well, but that can be done in a follow-up.

### CODE REVIEW

**Comment by bobsilverberg** [comment]
Requesting review from @willdurand as he and I were both working on this.

File: src/amo/pages/Home/index.js [hunk 1]
```diff
@@ -290,7 +282,6 @@ export class HomeBase extends React.Component {
             {_config.get('enableFeatureSponsoredShelf') && isDesktopSite ? (
               <PromotedAddonsCard
                 addonInstallSource={INSTALL_SOURCE_PROMOTED_SHELF}
-                addons={promotedExtensions}
                 loading={loading}
```
Comment (willdurand) [inline src/amo/pages/Home/index.js:285]: Maybe `loading` could be moved to `PromotedAddonsCard` too.

**Review by willdurand — APPROVED** [review]
I like it! Thanks, r+wc

File: src/amo/pages/Home/index.js (hunk 1 above)
Comment (bobsilverberg) [reply src/amo/pages/Home/index.js:285]: Good idea!

**Pull request merged**